README.md

```diff
@@ -1,145 +1,22 @@
-# Helm Chart for deployment of WSO2 API Microgateway 
+# Kubernetes and Helm Resources for WSO2 API Microgateway 
 
-## Contents
+*Helm Resources for container-based deployments of WSO2 API Microgateway.*
```

Comment:
And mention `Helm Resources for container-based deployments of WSO2 API Microgateway in Kubernetes`. WDYT?

Reply:
Done